File: LeetCode/1929-Concatenation-of-Array.cs
public class Solution {
    public int[] GetConcatenation(int[] nums) {
        int n = nums.Length;
        int[] Ans = new int[n * 2];

        for (int i = 0; i < n; i++)
        {
            Ans[i] = Ans[n + i] = nums[i];
        }
        return Ans;
    }
}

File: LeetCode/1929-Concatenation-of-Array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
const getConcatenation = nums => {
    let n = nums.length; 
    let ans = new Array(n * 2); 

    for(let i = 0; i < nums.length; i++){
        ans[i] = ans[nums.length + i] = nums[i];
    }

    return ans; 
};
